src/components/calltree/ProfileCallTreeView.js

```diff
@@ -16,7 +16,13 @@ type Props = {|
 |};
 
 const ProfileCallTreeView = (props: Props) => (
-  <div className="treeAndSidebarWrapper">
+  <div
+    className="treeAndSidebarWrapper"
+    id="calltree-tab"
```

Comment:
Maybe `calltreeTabPanelContent` to make it more describing? Also notice we try to use camel-cased identifiers most of the time.

(of course this comment applies to other slug belows)

Reply:
Hi @julienw since we went for the kebab case in the tabbar button, should this then be: calltree-tab-panel, flame-graph-tab-panel etc? I think we don't need to say "content" as the relationship "control & content" between the tab and tabpanel is already described in the aria-controls attribute.